gravitee-am-model/src/main/java/io/gravitee/am/model/I18nDictionary.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.model;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class I18nDictionary {
```

Comment:
I wondering if we should use directly this model obect in the REST API response :thinking:  ?
Even if we are not doing it for each object, I may be better to create a I18nDictionaryEntity into the ManagerApi module and make a mapping between the two? That way we can split the object manipulate by the Client/API from the object manipulate into the service layer, and we should be able to annotate the defaultLocale property with Jackson annotation to rename this attribute in "default" as defined into this issue https://github.com/gravitee-io/issues/issues/8067 

wdyt ?

Reply:
Hmm. In my view we have too many pojos as it is (e.g. NewDictionary, UpdateDictionary), I'd be loath to add another one which would add even greater complexity.